OpenRA.Game/FileFormats/WavLoader.cs

```diff
@@ -69,24 +68,17 @@ bool LoadSound(Stream s)
 						s.ReadBytes(FmtChunkSize - 16);
 						break;
 					case "fact":
-						{
-							var chunkSize = s.ReadInt32();
-							UncompressedSize = s.ReadInt32();
-							s.ReadBytes(chunkSize - 4);
-						}
-
+						var chunkSize = s.ReadInt32();
+						UncompressedSize = s.ReadInt32();
+						s.ReadBytes(chunkSize - 4);
 						break;
 					case "data":
 						DataSize = s.ReadInt32();
 						RawOutput = s.ReadBytes(DataSize);
 						break;
 					default:
-						// Ignore unknown chunks
-						{
-							var chunkSize = s.ReadInt32();
-							s.ReadBytes(chunkSize);
-						}
-
+						var unknownChunkSize = s.ReadInt32();
+						s.ReadBytes(unknownChunkSize);
```

Comment:
I use it to document what this is about. See the comment removal a few lines before.